docs/spaces/index.asciidoc

```diff
@@ -2,28 +2,112 @@
+* <<spaces-moving-objects, Move objects between spaces>>
+* <<spaces-delete-started, Disable the Spaces feature>>
 
 [float]
-[[spaces-getting-started]]
-=== Getting Started
+[[spaces-managing]]
+=== View, create, and delete spaces
+
+The **Management > Spaces** overview lists your spaces and provides actions 
+for you to create, edit, and delete spaces.
+
+[role="screenshot"]
+image::spaces/images/space-management.png["Space management"]
+
+[float]
+==== Create a space 
+
+You can create as many spaces as you like. Each space must have a unique 
+space identifier. This short text string becomes part of the 
+{kib} URL when you are inside that space. {kib} suggests a space identifier based 
+on the name of your space, but you can customize the identifier to your liking.
+You cannot change the space identifier once you create the space.
+
+{kib} also has an <<spaces-api, experimental API>> if you prefer to create spaces programatically.
+
+[float]
+==== Delete a space
+
+Deleting a space permanently removes the space and all of its contents. 
+You can't delete the default space, but you can customize it to your liking.
+
+[float]
+[[spaces-control-feature-visibility]]
+=== Control feature access based on user need
+
+You have control over which features are visible in each space. 
+For example, you might hide Dev Tools 
+in your "Executive" space or show Stack Monitoring only in your "Admin" space.
+Controlling feature 
+visibility is part of the add and edit workflow.
+
+Feature controls is not a security feature. To secure access 
+to specific features on a per-user basis, you must configure 
+<<xpack-security-authorization, Kibana Security>>.
+
+[role="screenshot"]
+image::spaces/images/edit-space-feature-visibility.png["Controlling features visiblity"]
+
+[float]
+[[spaces-control-user-access]]
+=== Control feature access based on user privilege
+
+When using Kibana with security, you can configure applications and features 
+based on your users’ privileges. This means different roles can have access 
+to different features in the same space. 
+Power users might have privileges to create and edit visualizations and dashboards, 
+while analysts or executives might have Dashboard and Canvas with read-only privileges.
+See <<adding_kibana_privileges>> for details.
+
+[role="screenshot"]
+image::spaces/images/spaces-roles.png["Controlling features visiblity"]
+
+[float]
+[[spaces-moving-objects]]
+=== Move saved objects between spaces
+Use {kib}'s <<managing-saved-objects-export-objects, import/export>> 
+interface to copy objects from one space to another.
+
+. Navigate to the space that contains your saved objects.
+. Export your saved objects via the import/export interface.
+. Navigate to the space where you want to import the objects.
+. Import your saved objects via the import/export interface.
+. (Optional) Delete the saved objects from the space you exported from, if you don't want to keep a copy there.
```

Comment:
This sentence is awkward. Maybe:

`(Optional) If you don't want to keep copies of the objects, delete them from the space.`